src/controllers/authController.js

```diff
@@ -88,7 +89,18 @@ const login = async (req, res) => {
   }
 };
 
+//logout endpoint
+
+const logout = async (req, res) => {
+  res.cookie('token', 'logout', {
```

Comment:
This works fine, it's setting the value of the `token` variable in the cookie to the string `"logout"` instead of whatever user data was there. But an even better way would be to just set the `token` to an empty string since that's more clear to anyone using the cookie that it's empty now. This can be done with `res.cookie('token', '', {...})` or even more simply with the [clearCookie](http://expressjs.com/en/5x/api.html#res.clearCookie) function: `res.clearCookie('token', {...})`

Reply:
done `const logout = async (req, res) => {
  res.clearCookie('token', {
    httpOnly: true,
    expires: new Date(Date.now() + 1000),
  });

  res.status(StatusCodes.OK).json({ msg: 'user logged out!' });
};`

Reply:
commit 7d57f4b4c85fdc0bd16394cb73ab7103c40eeeab